app/util/appUtil.js

```diff
@@ -252,6 +283,9 @@ async function runProcess(inputs, outputs) {
         .then((res) => {
           unsub = res
         })
+        .catch((err) => {
+          throw err
+        })
```

Comment:
I believe this is somewhat redundent but it might be worth adding a warn level log here. We shouldn't be getting extrinsics errors so this would be unexpected

Reply:
added